relayer-cli/src/commands/listen.rs

```diff
@@ -61,11 +61,12 @@ impl FromStr for EventFilter {
 #[derive(Debug, Parser)]
 pub struct ListenCmd {
     /// Identifier of the chain to listen for events from
+    #[clap(long = "chain", required = true, value_name = "CHAIN_ID")]
     chain_id: ChainId,
 
     /// Add an event type to listen for, can be repeated.
     /// Listen for all events by default (available: Tx, NewBlock).
-    #[clap(short = 'e', long = "event", value_name = "EVENT")]
+    #[clap(long = "event", value_name = "EVENT")]
```

Comment:
This is where the zeal to remove short options may be doing more harm than good.
Note that this is a repeating option, so it will be annoying to type it over and over again if there are many events to filter.

Reply:
Oh good catch, thank you!
I managed to find a solution, by adding the `multiple_values` it is not required to repeat the flag:

```
    /// Add an event type to listen for, can be repeated.
    /// Listen for all events by default (available: Tx, NewBlock).
    #[clap(long = "event", value_name = "EVENT", multiple_values = true)]
    events: Vec<EventFilter>,
```

This will have the following tests pass:

```
    #[test]
    fn test_listen_multiple_events() {
        assert_eq!(
            ListenCmd{ chain_id: ChainId::from_string("chain_id"), events: vec!(EventFilter::from_str("Tx").unwrap(), EventFilter::from_str("NewBlock").unwrap()) },
            ListenCmd::parse_from(&["test", "--chain", "chain_id", "--event", "Tx", "--event", "NewBlock"])
        )
    }

    #[test]
    fn test_listen_multiple_events_single_flag() {
        assert_eq!(
            ListenCmd{ chain_id: ChainId::from_string("chain_id"), events: vec!(EventFilter::from_str("Tx").unwrap(), EventFilter::from_str("NewBlock").unwrap()) },
            ListenCmd::parse_from(&["test", "--chain", "chain_id", "--event", "Tx", "NewBlock"])
        )
    }
```

I will update this with the other changes you suggested.

Reply:
I think there was a reason why we wanted it to have the `multiple_occurrences` behavior, i.e. every occurrence needs its own flag repetition. Most likely because in `multiple_values` style as above, additional values were easily confused with positional parameters. But now we don't have any positional parameters! @ancazamfir @mircea-c any opinions here?

That said, I'd still prefer most frequently and likely used options to also have short flags, and this seems to be one such case.

Reply:
Let's use `multiple_values` since, as you say, we don't have any positional arguments anymore.